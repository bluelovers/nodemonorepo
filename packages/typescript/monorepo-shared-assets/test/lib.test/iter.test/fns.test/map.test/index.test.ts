import {iter} from '../../../../../index'
const {range, map} = iter.fns

it('matches snapshot', () => {
  class ValueCarrier {
    readonly value: number

    constructor (value: number) {
      this.value = value
    }
  }

  expect(
    Array.from(
      map(range(16), x => new ValueCarrier(x))
    )
  ).toMatchSnapshot()
})
