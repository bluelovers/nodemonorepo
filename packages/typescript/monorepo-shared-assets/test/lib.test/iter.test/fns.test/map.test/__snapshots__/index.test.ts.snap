// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Array [
  ValueCarrier {
    "value": 0,
  },
  ValueCarrier {
    "value": 1,
  },
  ValueCarrier {
    "value": 2,
  },
  ValueCarrier {
    "value": 3,
  },
  ValueCarrier {
    "value": 4,
  },
  ValueCarrier {
    "value": 5,
  },
  ValueCarrier {
    "value": 6,
  },
  ValueCarrier {
    "value": 7,
  },
  ValueCarrier {
    "value": 8,
  },
  ValueCarrier {
    "value": 9,
  },
  ValueCarrier {
    "value": 10,
  },
  ValueCarrier {
    "value": 11,
  },
  ValueCarrier {
    "value": 12,
  },
  ValueCarrier {
    "value": 13,
  },
  ValueCarrier {
    "value": 14,
  },
  ValueCarrier {
    "value": 15,
  },
]
`;
